tools/halide_benchmark.h

```diff
@@ -1,40 +1,16 @@
 #ifndef BENCHMARK_H
 #define BENCHMARK_H
 
+#include <chrono>
 #include <limits>
 
+namespace Halide {
+namespace Tools {
+
 // Benchmark the operation 'op'. The number of iterations refers to
 // how many times the operation is run for each time measurement, the
 // result is the minimum over a number of samples runs. The result is the
 // amount of time in seconds for one iteration.
-#ifdef _WIN32
```

Comment:
Noting the change here - we are going to depend on <chrono> on windows after this change?